stable/dex-controller/templates/deployments.yaml

```diff
@@ -40,7 +40,6 @@ spec:
           {{- end }}
       - args:
         - --metrics-addr=127.0.0.1:8080
-        - --enable-leader-election
```

Comment:
Curious, why removing this?

Reply:
@samba see https://github.com/mesosphere/dex-controller/pull/32